### DISCUSSION TITLE
wUST transfer is missing/stuck

### DISCUSSION THREAD

**Discussion by ROOKSxx** [wormhole-foundation/wormhole#351] [Problems / bugs] [answered]
Hi, I've made a transfer of wUST from Ethereum to Solana and it got stuck on 'awaiting completion on Solana'. I've cleared cache and refreshed but the transaction doesn't show up in 'My Recent Transaction'.

here are the eth and sol tx hashes:
https://etherscan.io/tx/0x60a973c37bda678026d13e9166c213b87859ecb689c438277ac6ab48c1819ff6

https://explorer.solana.com/tx/4w1c9TJTss4V7Zv1QJ6Xi1qiRNqn7ctnctPZHYXSJ9XehnVEfkFrb9QA5zMPVSrXzwuyu77jdbWof3kSC3ChGpHR

Thanks 🙏

**Comment by leoluk** [comment, marked as answer]
Needed a little nudge, check again: https://explorer.solana.com/tx/5RxEVov1mh4GCTPZ9A79drwaCRTh3PKzTs5y8HX3Gt6eWXihtWsbABLcf9wqU1rndP8kWtXk6FxDehxroYkq5kd
